integration

```diff
@@ -743,6 +743,7 @@
   "RobertD502/home-assistant-petkit",
   "RobHofmann/HomeAssistant-GreeClimateComponent",
   "RobHofmann/HomeAssistant-PhilipsAndroid2014",
+  "robinostlund/homeassistant-svk-mimer",
```

Comment:
This seems to be abandoned already?
No movement since this PR was created, and several PRs waiting.

Reply:
It is still active 😊